sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> No option clash, but CSMA should be explicitly be enabled or disabled, not via CSMA_RETRIES. And what would NETCONF_OPT_CCA actually do? In this case AUTOCCA is more understandable.

Yes I agree, makes it more clear. Just thought you'd prefer the name NETCONF_OPT_CCA instead of AUTOCCA.

I don't really know how to proceed. (i) Using both options ..CCA and ..CSMA does not really fit to a practical implementation related to the atmel device, right? And (ii), Omitting the ..CCA option would fit to an implementation on the atmel device, but that would be very ambiguous when it comes to the KW2x device since it actually does not support CSMA.

What I need for KW2x device is an option that allows enabling the CCA part. The CSMA related options can not be implemented on the KW2x device anyways.